lib/elixir/src/elixir_erl.erl

```diff
@@ -22,10 +22,10 @@ debug_info(elixir_v1, _Module, none, _Opts) ->
 debug_info(elixir_v1, _Module, {elixir_v1, Map, _Specs}, _Opts) ->
   {ok, Map};
 debug_info(erlang_v1, _Module, {elixir_v1, Map, Specs}, _Opts) ->
-  {Prefix, Forms, _, _} = dynamic_form(Map),
+  {Prefix, Forms, _, _} = dynamic_form(Map, []),
```

Comment:
There are some problems with the protocol consolidation due to this. I'm still exploring how to solve this, but I think we need to add something to the tuple `{elixir_v1, Map, Specs}`, like `{elixir_v1, Map, Specs, Deprecated}` (and maybe bump the version), or add them to the `Map` itself. Otherwise, after consolidation `Module.__info__(:deprecated)` always return `[]`.

Another possibility is to send the extra chunks via `Opts` and recover the deprecated from them, but I'm not sure if this will work. I'll try out the different things that I have in mind to check.